INSTALL.md

```diff
@@ -1,95 +1,55 @@
-# Geokrety installation on a local filesystem
+# Geokrety installation
 
 If you note other requirements, feel free to add it here.
 
-# GeoKrety Website requirements
+## GeoKrety Website requirements
 
 * apache / apache2 + mod_rewrite
 * php
 * imagemagick
 * smarty 2 + smarty-gettext plugin
 
 
-# How to install GeoKrety locally using Docker Toolbox on Windows host
+## Run Geokrety locally
 
-## Requirements
+To run a geokrety instance on your Windows/Linux workstation using Docker :
 
-Please install the following products:
+ cf. [gkShell](gkShell/README.md)
 
-* Git for windows (aka 'git bash') : [download](https://gitforwindows.org/)
-* Docker Toolbox : [download](https://docs.docker.com/toolbox/toolbox_install_windows/)
-   * Add Docker toolbox installation directory to your PATH.
 
-Verification:
-* Open git bash; DockerToolbox should be available:
+## How to install contributors tools
 
-````
-    /C/Programmes/DockerToolbox/start.sh
-    docker-compose -version
-    docker-machine -version
-````
-
-## Steps
-
-### Check GeoKrety configuration
-
-* Apache configuration: [apache2](docker/apache2/)
-* MariaDB configuration(MySQL fork): [mariadb](docker/mariadb/)
-
-Create your own PHP configuration:
-
-* create `docker/configs/konfig-local.php` from `configs/konfig-local.tmpl.php`
-* create `docker/configs/konfig-mysql.php` from `configs/konfig-mysql.tmpl.php`
-* create `docker/configs/ssmtp.conf` from `configs/ssmtp.tmpl.conf`
-
-### Create GeoKrety docker-machine
-
-* create a local geokrety server ([custom](Dockerfile) [apache+php5](https://hub.docker.com/_/php/), [mariadb](https://hub.docker.com/_/mariadb/), [adminer](https://hub.docker.com/_/adminer/))
-
-````
-    ./install.sh
-````
-
-* the script will output you server ip and how to connect to your geokrety instance.
-
-
-### Update GeoKrety docker-machine content (website and configs)
+This section helps to execute locally some checks before pushing to github:
+- composer to retrieve dependencies,
+- Php Coding Standard Fixer,
+- PHPUnit.
 
-````
-    ./update.sh
-````
-
-### Update GeoKrety docker-machine one file
-
-````
-    ./update.sh onefile ruchy.php
-````
+For out-of-install scope contributors details, please see also [CONTRIBUTING.md](CONTRIBUTING.md)
 
-### Update GeoKrety docker-machine content (scripts)
+### Linux host
 
-Requirements: having done git clone of [geokrety-scripts repository](https://github.com/geokrety/geokrety-scripts) into `../geokrety-scripts/`
+Install php7:
 
-````
-    ./update.sh scripts
-````
+    sudo apt install php7.2-cli
+    php -version
 
-### Fix GeoKrety docker-machine directory/file rights (docker scp workaround)
+And php extension (for tests)
 
-````
-    ./update.sh rights
-````
+    sudo apt-get install php-fdomdocument
+    sudo apt-get install php-mbstring
```

Comment:
Debian/Ubuntu specific, and `php-fdomdocument` not available on Centos.. So finally we don't care about Centos ;)

Reply:
with section preambule, keep this as example (little reword the sentence)